### PR TITLE
MES-4442: Adding components for Cat A Mod 2 Test Details View

### PR DESCRIPTION
**Pull request by MarkPatt89**
## Description
- create test details view for Cat A Mod 2
- rename the file for Mod 1 component which referenced `BE`.
## Checklist

- [x] PR title includes the JIRA ticket number
- [x] Branch is rebased against the latest develop
- [x] Code has been tested manually
- [x] PR link added to JIRA ticket
- [x] One review from each scrum team
- [x] Squashed commit contains the JIRA ticket number

### CODE REVIEW

File: src/pages/view-test-result/cat-a-mod2/components/debrief-card/__tests__/debrief-card.spec.ts [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+import { FaultSummaryProvider } from '../../../../../../providers/fault-summary/fault-summary';
+import { FaultCountProvider } from '../../../../../../providers/fault-count/fault-count';
+// todo: PREP-AMOD2 change to CatAmod2UniqueTypes when schema changes are ready
+import { CatBEUniqueTypes } from '@dvsa/mes-test-schema/categories/BE';
+import {
+  DataRowListItem,
+  TestRequirementsLabels,
+  ViewTestResultLabels,
+} from '../../../../components/data-row-with-list/data-list-with-row.model';
+import { manoeuvreTypeLabels } from '../../../../../../shared/constants/competencies/catbe-manoeuvres';
```
Comment (szabikr) [inline src/pages/view-test-result/cat-a-mod2/components/debrief-card/__tests__/debrief-card.spec.ts:31]: A todo is missing for here

**Review by szabikr — APPROVED** [review]

File: src/pages/view-test-result/cat-a-mod2/components/debrief-card/__tests__/debrief-card.spec.ts (hunk 1 above)
Comment (MarkPatt89) [reply src/pages/view-test-result/cat-a-mod2/components/debrief-card/__tests__/debrief-card.spec.ts:31]: Thanks Szabi, this also helped highlight there were 2 missing `__tests__` folders I'd neglected.

File: src/pages/view-test-result/cat-a-mod2/__tests__/view-test-result.cat-a-mod2.page.spec.ts [hunk 2]
```diff
@@ -0,0 +1,275 @@
...
+import { ViewTestHeaderComponent } from '../../components/view-test-header/view-test-header';
+import { ViewTestHeaderModel } from '../../components/view-test-header/view-test-header.model';
+import { TestOutcome } from '../../../../modules/tests/tests.constants';
+import { DebriefCardComponent } from '../components/debrief-card/debrief-card';
+import { ErrorMessageComponent } from '../../../../components/common/error-message/error-message';
+import { BusinessDetailsCardComponent } from '../components/business-details-card/business-details-card';
+import { ContactDetailsCardComponent } from '../../components/contact-details-card/contact-details-card';
+import { ViewTestResultCatAMod2Page } from '../view-test-result.cat-a-mod2.page';
+
+describe('ViewTestResultCatBEPage', () => {
```
Comment (amobarak) [inline src/pages/view-test-result/cat-a-mod2/__tests__/view-test-result.cat-a-mod2.page.spec.ts]: Rename the describe please

**Comment by amobarak** [comment]
Just one small comment, otherwise looks good

**Review by amobarak — APPROVED** [review]

File: src/pages/view-test-result/cat-a-mod2/__tests__/view-test-result.cat-a-mod2.page.spec.ts (hunk 2 above)
Comment (MarkPatt89) [reply src/pages/view-test-result/cat-a-mod2/__tests__/view-test-result.cat-a-mod2.page.spec.ts]: Fixed thanks, Ahmed.

**Pull request merged**